src/components/mosaic-gallery/MosaicGalleryItem.tsx

```diff
@@ -49,6 +50,8 @@ const MosaicGalleryItem: React.FC<IMosaicGalleryItemProps> = ({
     titleFontFamily,
     titleFontSize,
   } = mosaicSettings as IMosaicSettings;
+  const imageBorderRadius =
+    padding + 15 < borderRadius ? borderRadius - padding : borderRadius / 2;
```

Comment:
the logic is incorrect when border radius is lower than 15: e.g. borderRadius: 10px, padding: 3px, in this case inner radius should be 7px, but with actual logic it is 5x

Reply:
changed

Reply:
with this logic you are not covering the case when the radius is bigger 1-5px, e.g. padding 20px, radius 21px

Reply:
the working condition is this: `padding < borderRadius / 2 ? borderRadius - padding : borderRadius / 2;`